extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,127 @@
+
+- Examples: [Tiled](examples/example-tiled.json), [Dimension](examples/example-dimension.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item properties
+
+| Field Name        | Type                                                                                   | Description                                                                                                     |
+| ----------------- | -------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| tl:tileMatrixSet  | [[TileMatrixSet Object](#tile-matrix-set-object)]                                      | An array of available tile matrix sets                                                                          |
+| tl:dimensions     | Map<string, [Discrete Dimension Object](#discrete-dimension-object)\|[number\|string]> | Additional dimensions. The keys of this object can be used to replace the template parameters of the same name. |
+
+**tl:dimension:** The possible values of this dimension can be expressed as either a finite list of values (using an array of values) or as a discrete dimension using the [Discrete Dimension Object](#discrete-dimension-object).
+
+### Tile Matrix Set Object
+
+The description of the tile pyramid.
+
+| Field Name        | Type                                               | Description                                                                                                                  |
+| ----------------- | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| type              | string                                             | **REQUIRED.** Type of the Tile Matrix Set. Fixed to `TileMatrixSetType`                                                      |
+| title             | string                                             | Title of this tile matrix set, normally used for display to a human                                                          |
+| abstract          | string                                             | Brief narrative description of this tile matrix set, normally available for display to a human                               |
+| keywords          | [string]                                           | Unordered list of one or more commonly used or formalized word(s) or phrase(s) used to describe this dataset                 |
+| identifier        | string                                             | **REQUIRED.** Tile matrix set identifier. This can be used as the `{TileMatrixSet}` template parameter.                      |
+| boundingBox       | [BoundingBox Object](#bounding-box-object)         | Minimum bounding rectangle surrounding the tile matrix set, in the supported CRS                                             |
```

Comment:
Isn`t the bounding box already specified in the item?

Reply:
In the latest version there is no bounding box definition in the extension spec anymore. For inline tile matrix definitions the bounding box from the 2D TMS schema can still be used.